devicetypes/ezex/somfy-blind-controller-ezex.src/somfy-blind-controller-ezex.groovy

```diff
@@ -0,0 +1,123 @@
+/**
+ *
+ *	Copyright 2019 eZEX
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition(name: "SOMFY Blind Controller/eZEX", namespace: "eZEX", author: "eZEX Corp", vid:"SmartThings-smartthings-Somfy_WindowShade", ocfDeviceType: "oic.d.blind") {
```

Comment:
@lukhong @greens @dkirker I think this DTH could be potentially reused for other Zigbee shades. Thoughts on renaming/moving it to `Zigbee Window Shade`? We already have a `Z-Wave Window Shade`.

Reply:
I think so if possible.

Reply:
I'm reading over it, but I imagine that we could very easily.

Reply:
I think we should rename it then. There is nothing device specific in here so it might be useable for other similar devices.

Reply:
Yeah, then I can push a commit that changes the name and the file path.
Then the vid should be changed to `generic-shade`, right? But it needs to be updated because it shows as 'dimming level' not the 'shade level'. Can @dkirker change the `generic-shade` in order to be fit to shade device?

Reply:
Oh, `vid:` is working in the fingerprint! I will make the commit for this change.

Reply:
`generic-shade` can be updated (and should be).

Reply:
@dkirker , I want to know if you are able to update `generic-shade` and merge this PR before next Tuesday so that the certification step start during next week?